src/app/components/Button/Button.module.css

```diff
@@ -0,0 +1,28 @@
+.submit {
+  background-color: var(--main-font-color);
+  color: var(--secondary-font-color);
+  font: var(--main-font);
+  border: none;
+  border-radius: 0.25em;
+  padding: 1em 5em;
+}
+
+.submit:active {
+  background-color: var(--addButton-active);
+}
+
+.delete {
+  background-color: var(--hint-color);
+  border: none;
+  border-radius: 0.25em;
+  padding: 0.5em 1.5em;
+  font: var(--main-font);
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
```

Comment:
flexbox supports `gap` since Chromium v83. Older browsers will not display the gap. If this is an issue, fallback to grid or add a margin.
https://developer.mozilla.org/en-US/docs/Web/CSS/gap#support_in_flex_layout